.github/workflows/build_deploy_docs.yml

```
@@ -18,6 +18,13 @@ jobs:
             5.0.x
             6.0.x
 
+      - uses: actions/cache@v3
+        with:
+          path: ~/.nuget/packages
+          key: ${{ runner.os }}-nuget-${{ hashFiles('**/*.csproj') }}
```

What about F# project files? Does this glob work?
```suggestion
          key: ${{ runner.os }}-nuget-${{ hashFiles('**/*.*sproj') }}
```